optimum/intel/openvino/quantization.py

```diff
@@ -409,25 +407,25 @@ def _quantize_torchmodel(
                 data_collator=data_collator,
             )
 
-            model_inputs = next(iter(calibration_dataloader))
-            ov_config.add_input_info(model_inputs)
-            nncf_config = NNCFConfig.from_dict(ov_config.__dict__)
-            nncf_config = register_default_init_args(nncf_config, calibration_dataloader)
-            controller, model = create_compressed_model(
```

Comment:
if create_compressed_model is deprecated, we should also update the trainer : 

https://github.com/huggingface/optimum-intel/blob/c2d267aef2fc2185be8cece275fc27e13f9b3602/optimum/intel/openvino/trainer.py#L259

Reply:
This unfortunately can't be replaced, because in the trainer `create_compressed_model()` is used for other compression methods besides quantization. 

As discussed with @AlexKoff88, we plan to keep `create_compressed_model()` in the OVTrainer for now. But to refactor the way config is provided to it.

Reply:
got it !